ubuntu/debian/control

```diff
@@ -250,3 +251,34 @@ Description: Collection of useful code used by robotics apps - Profiler dev file
  parsing 3D mesh files, managing console output, and using PID controllers.
  .
  Profiler development files
+
+Package: libignition-common5-testing
+Architecture: any
+Section: libs
+Pre-Depends: ${misc:Pre-Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}
+Multi-Arch: same
+Description: Collection of useful code used by robotics apps - Testing libries
+ Ignition common is a component in the Ignition framework, a set of
+ libraries designed to rapidly develop robot applications. A collection of
+ useful classes and functions for handling many command tasks. This includes
+ parsing 3D mesh files, managing console output, and using PID controllers.
+ .
+ Testing component of the library
+
+Package: libignition-common5-testing-dev
+Architecture: any
+Section: libdevel
+Depends: libgdal-dev,
```

Comment:
I think it shouldn't depend on `libgdal-dev`

Reply:
Agree, that is strange.

Reply:
Bad copy-paste, fixed in [d9db5d8](https://github.com/ignition-release/ign-common5-release/pull/10/commits/d9db5d8b970c6f80e6de37b8626b04c3069785c2)